.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+    - name: Generate CLI Build
+      run: |
+          go build -o dist/crda
+    - name: Archive production artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: crda
+        path: |
+          dist
```

Comment:
Why this is required? Isn't `goreleaser/goreleaser-action` does the building and uploading?

Reply:
Ack